mitosheet/mitosheet/errors.py

```diff
@@ -507,6 +507,19 @@ def make_invalid_simple_import_error(error_modal: bool=False) -> MitoError:
         error_modal=error_modal
     )
 
+def make_invalid_snowflake_import_error() -> MitoError:
+    """
+    Helper function for creating an invalid_snowflake_import_error.
+
+    Occures when the user tries to simple import and it fails.
+    """
+    return MitoError(
+        'invalid_snowflake_import_error',
+        'Cannot Query Database',
+        'We were unable to execute that query. Reach out to support for help.',
```

Comment:
When we do more testing, let's figure out if the error message that Snowflake returns is helpful and maybe display that to the user here.